app/src/main/AndroidManifest.xml

```diff
@@ -50,6 +50,7 @@
             android:theme="@style/AppTheme.NoActionBar" />
         <activity
             android:name=".activities.BluetoothConnectionActivity"
+            android:launchMode = "singleInstance"
```

Comment:
Well, actually... I want you to remove this 😅 

If you had taken just 1 minute of your time and looked at the code in the `MainActivity`, you would have seen that the activity is simply started twice (also applies to the NFC activity)... See [here](https://github.com/ericcornelissen/NervousFish/blob/develop/app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java#L252)

I will open a separate PR with a fix for this 😄